Source/RMG/UserInterface/Widget/RomBrowserWidget.cpp

```diff
@@ -535,6 +556,32 @@ void RomBrowserWidget::on_listViewWidget_sectionMoved(int logicalIndex, int oldV
     CoreSettingsSetValue(SettingsID::RomBrowser_ColumnOrder, columnOrder);
 }
 
+void RomBrowserWidget::on_listViewWidget_headerContextMenuRequested(QPoint position)
```

Comment:
This function should be removed

Reply:
Done, will include in next commit. (See my comment on your fourth requested change).